### PR TITLE
Chat bad connection notification moved up

### PR DESCRIPTION
**Pull request by DreamySkrell**
image

  - tweak: "Chat bad connection notification/popup moved up so it does not block new messages in the chat."

### CODE REVIEW

**Comment by DreamySkrell** [comment]
!review

File: tgui/packages/tgui-panel/styles/components/Notifications.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 .Notifications {
   position: absolute;
-  bottom: 1em;
+  bottom: 10em;
```
Comment (SleepyGemmy) [inline tgui/packages/tgui-panel/styles/components/Notifications.scss:8]:
```suggestion
  top: 1em;
```
probably?

**Review by SleepyGemmy — COMMENTED** [review]
this looks worse than before. consider making it align to the top of the chatbox.

File: tgui/packages/tgui-panel/styles/components/Notifications.scss (hunk 1 above)
Comment (DreamySkrell) [reply tgui/packages/tgui-panel/styles/components/Notifications.scss]: Indeed.

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**